app/events/db/migrations/db_table_sql_scripts.sql

```diff
@@ -864,3 +864,125 @@ CREATE TABLE IF NOT EXISTS iiq_schema.professional_contents_fetch_request
     CONSTRAINT        unique_professional_contents_fetch_request UNIQUE (iiq_id)
 );
 ALTER TABLE iiq_schema.professional_contents_fetch_request OWNER TO iiq;
+
+CREATE TABLE IF NOT EXISTS iiq_schema.profiles_search_export_data
```

Comment:
when professional comes, we won't have a way to distinguish
```suggestion
CREATE TABLE IF NOT EXISTS iiq_schema.social_profiles_search_export_data
```